.gitignore

```diff
@@ -1,5 +1,6 @@
 # OS
 .DS_Store
+package-lock.json
```

Comment:
Just checking that this was supposed to be here? package-lock.json needs to be committed: 
https://docs.npmjs.com/cli/v10/configuring-npm/package-lock-json#description

Reply:
Oh ignore that you removed it as i was typing it